front/app/containers/Admin/pagesAndMenu/containers/ProjectsList/ProjectsListContent.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import { List, Row } from 'components/admin/ResourceList';
+import ProjectRow from 'containers/Admin/projects/components/ProjectRow';
+import Link from 'utils/cl-router/Link';
+import Warning from 'components/UI/Warning';
+
+// hooks
+import useAdminPublications from 'hooks/useAdminPublications';
+import useCustomPage from 'hooks/useCustomPage';
+import { useParams } from 'react-router-dom';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { adminCustomPageSettingsPath } from '../../routes';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+const ProjectsListContent = ({
+  areaIds,
+  topicIds,
+}: {
+  areaIds: string[];
+  topicIds: string[];
+}) => {
+  const { list: adminPublications } = useAdminPublications({
+    topicFilter: topicIds,
+    areaFilter: areaIds,
+    publicationStatusFilter: ['published'],
+    rootLevelOnly: true,
```

Comment:
We will also show archived projects & projects inside folders. I use this for Events:

```
  const adminPublications = useAdminPublications({
    topicIds,
    areaIds,
    publicationStatusFilter: ['published', 'archived'],
  });
```

Reply:
Could you add a comment here + in CustomPageEvents.tsx that they should be the same?

Reply:
Or even better: you can export the parameters to the CustomPageEvents file.